build/Common.test.props

```diff
@@ -15,4 +20,28 @@
     <AdditionalFiles Include="$(MSBuildThisFileDirectory)/stylecop.json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+  </ItemGroup>
+
+  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
+    <IsTestProject>true</IsTestProject>
+    <CollectCoverage Condition="$(CollectCoverage) == ''">true</CollectCoverage>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <CollectCoverage Condition="$(CollectCoverage) == '' OR !$(TargetFramework.StartsWith('netcoreapp'))">false</CollectCoverage>
+  </PropertyGroup>
+
+  <PropertyGroup Label="CollectCodeCoverageResults" Condition="$(CollectCoverage)">
+    <CoverletOutputFormat>cobertura</CoverletOutputFormat>
+    <_BaseDir>..\..\TestResults\</_BaseDir>
+    <CoverletBaseDir>$(_BaseDir)Coverage\</CoverletBaseDir>
+    <CoverletResultsDir>$(_BaseDir)Results\</CoverletResultsDir>
+    <CoverletBaseDir Condition="$([MSBuild]::IsOSUnixLike())">$(CoverletBaseDir.Replace('\', '/'))</CoverletBaseDir>
+    <CoverletResultsDir Condition="$([MSBuild]::IsOSUnixLike())">$(CoverletResultsDir.Replace('\', '/'))</CoverletResultsDir>
+    <CoverletOutput>$(CoverletBaseDir)$(MSBuildProjectName).xml</CoverletOutput>
+    <Exclude>[xunit*]*,[*.Tests]*</Exclude>
+    <DebugType>pdbonly</DebugType> <!-- Coverlet needs full debug symbols -->
```

Comment:
you only enable full symbols for test project, correct? Looking at condition `Condition="$(CollectCoverage)"`. Would it show proper coverage for main libraries?

Reply:
So the symbols confused me a bit in the project as only in debug mode were symbols being generated as full. Is it worth unifying this to be all the time? If so, then I can remove this line.

Reply:
I think we wanted portable for linux. I vaguely remember there were some issues with full pdb ross platform. We can switch to full and than revert if issues would be discovered